presto-raptor/src/test/java/com/facebook/presto/raptor/storage/organization/TestCompactionSetCreator.java

```diff
@@ -103,22 +106,53 @@ public void testNonTemporalRowCountBasedOrganizationSet()
     public void testTemporalCompactionNoCompactionAcrossDays()
             throws Exception
     {
-        long day1 = Duration.ofDays(Duration.ofNanos(System.nanoTime()).toDays()).toMillis();
-        long day2 = Duration.ofDays(Duration.ofMillis(day1).toDays() + 1).toMillis();
-        long day3 = Duration.ofDays(Duration.ofMillis(day1).toDays() + 2).toMillis();
+        DateTimeZone localTimeZone = DateTimeZone.forTimeZone(TimeZone.getTimeZone("PST"));
```

Comment:
Does this need to match the time zone used to create `CompactionSetCreator`? If so, make it a constant